extra-radio/qsstv/autobuild/patch

```diff
@@ -0,0 +1,2 @@
+abinfo 'Patching the qsstv.pro: rename qwt-qt5 to qwt after LIBS variale...'
+sed -i 's/LIBS += \-lqwt\-qt5/LIBS += \-lqwt/g' "$SRCDIR"/qsstv.pro
```

Comment:
I'd suggest that you make your `sed` commands clearer, use full `-e` and `-i` expressions, and wrap long lines. Here's how I'd write this:

```
sed -e 's/LIBS += \-lqwt\-qt5/LIBS += \-lqwt/g' \
    -i "$SRCDIR"/qsstv.pro
```

Reply:
I referenced this from other packages' autobuild/patch, most of them use one-line command. But obviously this suggestion looks more reasonable.